libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -436,27 +436,54 @@ void AP_TECS::_update_speed_demand(void)
     // Use 50% of maximum energy rate to allow margin for total energy contgroller
     const float velRateMax = 0.5f * _STEdot_max / _TAS_state;
     const float velRateMin = 0.5f * _STEdot_min / _TAS_state;
+    const float vel_jerk = 0.6f;
```

Comment:
This might be too slow for some applications. For example if i look at the throttle response in your examples, a faster throttle ramp with a small amount (say 5%) overshoot would be preferred. It might have to be tuneable via a parameter.

Reply:
I've made it a parameter. But really I think the default is fast enough. I think I had it lower during the test I got the data in the description from